### DISCUSSION TITLE
Helping out with the latest code on github

### DISCUSSION THREAD

**Discussion by erikdubois** [archlinux/archinstall#3013] [General]
2024/12/08 or this commit [4592280](https://github.com/archlinux/archinstall/commit/4592280baaa6fc38ecad29f08290b7759429fb83)

We have now this screen in the mirrors

![image](https://github.com/user-attachments/assets/dba93f17-17d0-4937-ae86-4e4018d6adbb)

When pressing enter we get this.

![image](https://github.com/user-attachments/assets/3c53fe59-6419-450d-b2e1-e956e554487e)

We are missing a "/" in the url there. 

Lets fix that and see what we get.

**Comment by codefiles** [comment]
This should be an issue, not a discussion.

**Comment by correctmost** [comment]
The empty region issue is covered by https://github.com/archlinux/archinstall/issues/3081.

The URL issue is covered by https://github.com/archlinux/archinstall/pull/3109.

**Comment by erikdubois** [comment]
tested it with the code from this commit 

https://github.com/erikdubois/archinstall/commit/c0a2de4330f6a1931a59f0a38cd17daea3a7920d

After building the iso with your build script this is what I got.

![image](https://github.com/user-attachments/assets/a2621540-cdb1-4486-8560-503228040139)

and here is what I did

https://youtu.be/NhzV6cvgpBU

**Reply by correctmost** [reply]
The fixes for https://github.com/archlinux/archinstall/issues/3081 are still in progress and not on master yet :)

**Reply by erikdubois** [reply]
ok that makes sense then
too quick

You have a tester when you let me know.

**Comment by erikdubois** [comment]
everything ok in todays release
